subsys/bluetooth/host/classic/hfp_hf.c

```diff
@@ -1410,6 +1410,47 @@ int bt_hfp_hf_accept(struct bt_conn *conn)
 	return err;
 }
 
+static int chup_finish(struct at_client *hf_at, enum at_result result,
+		   enum at_cme cme_err)
+{
+	struct bt_hfp_hf *hf = CONTAINER_OF(hf_at, struct bt_hfp_hf, at);
+
+	LOG_DBG("AT+CHUP (result %d) on %p", result, hf);
+
+	return 0;
+}
+
+int bt_hfp_hf_reject(struct bt_conn *conn)
+{
+	struct bt_hfp_hf *hf;
+	int err;
+
+	LOG_DBG("");
+
+	if (!conn) {
+		LOG_ERR("Invalid connection");
+		return -ENOTCONN;
+	}
+
+	hf = bt_hfp_hf_lookup_bt_conn(conn);
+	if (!hf) {
+		LOG_ERR("No HF connection found");
+		return -ENOTCONN;
+	}
+
+	if (!atomic_test_bit(hf->flags, BT_HFP_HF_FLAG_INCOMING)) {
+		LOG_ERR("No incoming call setup in progress");
+		return -EINVAL;
+	}
+
+	err = hfp_hf_send_cmd(hf, NULL, chup_finish, "AT+CHUP");
+	if (err < 0) {
```

Comment:
In the previous commit you're comparing the return value from `hfp_ag_send_data()` as `(err != 0)`. Here you're doing `(err < 0)`. Does the return value really need to be treated differently? Generally, the most intuitive form to me is just `if (err) {`, but most important is to be consistent with whatever style you choose.

Reply:
Updated.